Lib/test/test_httpservers.py

```diff
@@ -1535,6 +1537,197 @@ def test_server_test_ipv4(self, _):
             server.test(ServerClass=mock_server, bind=bind)
             self.assertEqual(mock_server.address_family, socket.AF_INET)
 
+@mock.patch('http.server.test')
```

Comment:
I'd prefer having a test-by-test mocking rather than an entire mock active on the class.